docs/pages/2020_BIGMAP/sections/qe.rst

```diff
@@ -95,12 +118,113 @@ Download the :download:`demo_calcjob.py <include/snippets/demo_calcjob.py>` scri
         $ verdi import -n http://legacy-archive.materialscloud.org/file/2018.0001/v3/SSSP_efficiency_pseudos.aiida
         $ verdi import -n http://legacy-archive.materialscloud.org/file/2018.0001/v3/SSSP_precision_pseudos.aiida
 
-Finally, submit the calculation using:
+Along with the PK of the ``StructureData`` node for the silicon structure we imported in the previous section, we now have everything to set up the calculation step by step.
+We will do this in the ``verdi shell``, an interactive IPython shell that has many basic AiiDA classes pre-loaded.
+To start the IPython shell, simply type in the terminal:
 
 .. code-block:: console
 
-    $ verdi run demo_calcjob.py
-    Submitted CalcJob with PK=179
+    $ verdi shell
+
+First, we'll load the code from the database using its label:
+
+.. code-block:: ipython
+
+    In [1]: code = load_code(label='pw')
+
+Every code has a convenient tool for setting up the required input, called the *builder*.
+It can be obtained by using the ``get_builder`` method:
+
+.. code-block:: ipython
+
+    In [2]: builder = code.get_builder()
+
+Let's supply the builder with the structure we just imported.
+Replace the ``<STRUCTURE PK>`` with that of the structure we imported at the start of the section:
+
+.. code-block:: ipython
+
+    In [3]: structure = load_node(<STRUCTURE PK>)
+       ...: builder.structure = structure
```

Comment:
I think in all the rest of the tutorial we use underscore? At least I do and you do below in `<PSEUDO_FAMILY>`

```suggestion
Let's supply the builder with the structure we just imported.
Replace the ``<STRUCTURE_PK>`` with that of the structure we imported at the start of the section:

.. code-block:: ipython

    In [3]: structure = load_node(<STRUCTURE_PK>)
       ...: builder.structure = structure
```